src/biogeochem/CNCIsoFluxMod.F90

```diff
@@ -1150,13 +1148,12 @@ subroutine CIsoFlux3(num_soilc , filter_soilc, num_soilp  , filter_soilp, &
                           +iso_cnveg_cf%m_deadcrootc_storage_to_litter_fire_patch(pp) + &
                           iso_cnveg_cf%m_deadcrootc_xfer_to_litter_fire_patch(pp))* croot_prof(pp,j)) * patch%wtcol(pp)    
                      
-                     iso_cnveg_cf%m_c_to_litr_cel_fire_col(cc,j) = iso_cnveg_cf%m_c_to_litr_cel_fire_col(cc,j) + &
-                          (iso_cnveg_cf%m_leafc_to_litter_fire_patch(pp)*lf_fcel(ivt(pp))*leaf_prof(pp,j) + &
-                          iso_cnveg_cf%m_frootc_to_litter_fire_patch(pp)*fr_fcel(ivt(pp))*froot_prof(pp,j)) * patch%wtcol(pp) 
-                     
-                     iso_cnveg_cf%m_c_to_litr_lig_fire_col(cc,j) = iso_cnveg_cf%m_c_to_litr_lig_fire_col(cc,j) + &
-                          (iso_cnveg_cf%m_leafc_to_litter_fire_patch(pp)*lf_flig(ivt(pp))*leaf_prof(pp,j) + &
-                          iso_cnveg_cf%m_frootc_to_litter_fire_patch(pp)*fr_flig(ivt(pp))*froot_prof(pp,j)) * patch%wtcol(pp)  
+                     do i = i_litr_min+1, i_litr_max
+                        iso_cnveg_cf%m_c_to_litr_fire_col(cc,j,i) = &
+                           iso_cnveg_cf%m_c_to_litr_fire_col(cc,j,i) + &
+                           (iso_cnveg_cf%m_leafc_to_litter_fire_patch(pp) * lf_f(ivt(pp),i) * leaf_prof(pp,j) + &
+                           iso_cnveg_cf%m_frootc_to_litter_fire_patch(pp) * fr_f(ivt(pp),i) * froot_prof(pp,j)) * patch%wtcol(pp)
+                     end do
```

Comment:
These are the remaining litter pools handled in a loop. 
Before the code expected three litter pools. 
Now the code expects a minimum of two but any maximum.

Reply:
Adding a comment to that effect in clm_varpar.